docs/adrs/0003-bulk-import-ui.md

```diff
@@ -0,0 +1,98 @@
+# Bulk Import UI
+
+Date: 2020-10-26
+
+A primary goal of the current data upload refactoring project includes enabling users to upload data in bulk and producing multiple new data objects with only a few user-side operations. That means introducing an interface that can handle the following steps:
```

Comment:
A link to the figma designs, or else some screenshots would be helpful here for context for readers less familiar with the project